documentation.html

```diff
@@ -768,8 +771,281 @@ <h6 class="text-regular-weight product-title">
         </div>
         <br />
         <hr />
+
+        <!--Input form-->
+        <div id="Input">
+          <div class="component-intro">
+            <h1>Input</h1>
+            <p class="body-cp-lg text-light-weight">
+              There are form fields listed below and form validation is also
+              styled.
+            </p>
+          </div>
+
+          <div class="component-display-container">
+            <div class="row">
+              <label class="form-label form-label-required-field"
+                >Email address</label
+              >
+              <input
+                type="email"
+                class="form-field"
+                placeholder="name@example.com"
+                required
+              />
+            </div>
+            <div class="row">
+              <label class="form-label">Example Textarea</label>
+              <textarea
+                type="email"
+                class="form-field form-textarea"
+                placeholder="start writing here.."
+                rows="3"
+              ></textarea>
+            </div>
+          </div>
+
+          <div class="code-snippet-display">
+            <iframe
+              src="https://www.thiscodeworks.com/embed/620ea15b14fadf0015661d29"
+              style="width: 100%; height: 216px"
+              frameborder="0"
+            ></iframe>
+          </div>
+
+          <div class="component-display-container">
+            <h2 class="h4">Disabled Input</h2>
```

Comment:
h2 element and h4 class to it looks a little off. Dont keep class names same as that off already existing html elements